src/search/css/SearchForm.scss

```diff
@@ -1,3 +1,27 @@
-form {
-  background: green;
+@import "inc/vars";
+
+.search-form {
+  display: flex;
+
+  label {
+    font-size: $font-size-m;
+  }
+
+  input {
+    box-sizing: border-box !important;
+    flex-grow: 1;
+    font-size: $font-size-l;
+    height: 3rem;
```

Comment:
Using `em` and `rem` sounds like a good idea but every time I look at something describing how to use them it feels way too complicated. MDN linked to [*Compose to a Vertical Rhythm*](https://24ways.org/2006/compose-to-a-vertical-rhythm) and putting that math in feels like it could be error prone to me.

I feel like we should have some mixins for setting font size while maintaining line-height, etc. If that were the case I don't think we'd need to set an explicit height here (but I could quite definitely be wrong, I've never used `em`/`rem`).